doc/gallery/plot_rasterio_rgb.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+.. _recipes.rasterio_rgb:
+
+============================
+imshow() and map projections
+============================
+
+Using rasterio's projection information for more accurate plots.
+
+This example extends :ref:`recipes.rasterio` and plots the image in the
+original map projection instead of relying on pcolormesh and a map
+transformation.
+"""
+
+import os
+import urllib.request
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+# Download the file from rasterio's repository
+url = 'https://github.com/mapbox/rasterio/raw/master/tests/data/RGB.byte.tif'
+urllib.request.urlretrieve(url, 'RGB.byte.tif')
+
+# Read the data
+da = xr.open_rasterio('RGB.byte.tif')
+
+# Normalize the image
+da = da / 255
```

Comment:
I don't think this is strictly necessary with RGB support in imshow, though I suppose it doesn't hurt.

Reply:
That's unfortunately necessary, see https://github.com/pydata/xarray/issues/1880